inertia/http.py

```diff
@@ -18,20 +18,21 @@
 INERTIA_SSR_TEMPLATE = "inertia_ssr.html"
 
 
-class InertiaRequest:
+class InertiaRequest(HttpRequest):
     def __init__(self, request):
-        self.request = request
-
-    def __getattr__(self, name):
-        return getattr(self.request, name)
+        super().__init__()
+        self.__dict__.update(request.__dict__)
 
     @property
     def headers(self):
-        return self.request.headers
+        return super().headers
 
     @property
     def inertia(self):
-        return self.request.inertia.all() if hasattr(self.request, "inertia") else {}
+        inertia_attr = self.__dict__.get("inertia")
+        return (
+            inertia_attr.all() if inertia_attr and hasattr(inertia_attr, "all") else {}
```

Comment:
It looks like the refactor equivalent should be something like

```python
return self.inertia.all() if hasattr(self, "inertia") else {}
```

does that look right?

Reply:
ahhh yeah I get that. Ok, I think this makes sense and maybe we'll come back to it later. Thanks for double checking!